internal/pkg/pkg.go

```diff
@@ -695,3 +699,49 @@ func SetPkgPathAnnotation(rn *yaml.RNode, pkgPath types.UniquePath) error {
 func RemovePkgPathAnnotation(rn *yaml.RNode) error {
 	return rn.PipeE(yaml.ClearAnnotation(pkgPathAnnotation))
 }
+
+// RGFile returns the resourcegroup object by lazy loading it from the filesytem.
+func (p *Pkg) RGFile(filename string) (*rgfilev1alpha1.ResourceGroup, error) {
+	if p.rgFile == nil {
+		// TODO(rquitales): Handle real reading errors vs file does not exist.
+		rg, _ := ReadRGFile(p.UniquePath.String(), filename)
+		p.rgFile = rg
+	}
+	return p.rgFile, nil
+}
+
+// TODO(rquitales): Create new error types for resourcegroup.
+func ReadRGFile(p, filename string) (*rgfilev1alpha1.ResourceGroup, error) {
```

Comment:
Suggest to change these as private functions (DecodeRGFile).

Reply:
for public functions, variable names should be bit descriptive, so `p` is not helping :) 
I might have suggestion on potential location for these methods after I have good idea of the code here.

Reply:
The `ReadRGFile()` function is also used within the cmdliveinit_test.go file, so it needs to be exported. This is similar to the existing `ReadKptfile()` function as well. Lmk if there is a preference to refactor the code to make this cleaner.